account_financial_report/report/general_ledger.py

```diff
@@ -51,7 +51,12 @@ def _get_taxes_data(self, taxes_ids):
         return taxes_data
 
     def _get_account_internal_types(self, grouped_by):
-        return ["receivable", "payable"] if grouped_by != "taxes" else ["other"]
+        if grouped_by == "partners":
+            return ["receivable", "payable"]
+        elif grouped_by == "taxes":
+            return ["other"]
+        else:
+            return ["receivable", "payable", "other", "liquidity"]
```

Comment:
Can you better return `[]` and change the code where this is used for avoiding that if any new type, this won't work?

Reply:
I changed the fix a bit. The issue around new types should be good now